init.d.misc/wpa_supplicant.in

```diff
@@ -39,6 +39,11 @@ find_wireless()
 			fi
 		done
 		;;
+	FreeBSD)
+		for iface in $(sysctl -b net.wlan.devices) 2>/dev/null`; do
```

Comment:
Errant backtick here, where the right parenthesis should be.  (ie, the line should look like this:

```sh
for iface in $(sysctl -b net.wlan.devices 2>/dev/null); do
```